### ISSUE TITLE
Search result for CSRF

### ISSUE THREAD

**Issue by tkrpata** [TeamMentor/TM_4_0_Design#762]
Should I put this here or in TM Content? (If the latter, how to get access?)

Te first search result for search term "csrf" probably should be:
https://beta.teammentor.net/article/7b54c2df9e5a/Cross-Site-Request-Forgery
but is
https://beta.teammentor.net/article/ba3c65a62479/How-to-Test-for-Cross-Site-Request-Forgery-(CSRF)-Vulnerabilities

**Comment by roman87** [comment]
@tkrpata this should be in this repo. Its a tricky one, but this has more to do with the algorithm of the search rather than the content. 
Anyways I also sent you an invite for the TMContent org which will give you access to 
https://github.com/TMContent/Lib_UNO

**Comment by roman87** [comment]
The first thing to do would be to analyze the scores. Which I will do.

**Comment by DinisCruz** [comment]
Yeah, so part of the prob is that that article does not have CSRF on the title and there are not that many mentions of it in it

![image](https://cloud.githubusercontent.com/assets/656739/7601304/36c08d80-f90c-11e4-87a7-e95121a9069e.png)

specially when compared with 

![image](https://cloud.githubusercontent.com/assets/656739/7601311/5067b240-f90c-11e4-81c2-6a074ab427e6.png)

I think an interesting what to solve this is to give more **points** to Issues that are **Vulnerability** and **Any Technology**

![image](https://cloud.githubusercontent.com/assets/656739/7601339/8a295510-f90c-11e4-84dc-c32145293c99.png)

**Comment by roman87** [comment]
The real problem is that we do not treat CSRF, Cross-Site Request Forgery, and Cross Site Request Forgery as the same thing. Which we should.
